Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -80,6 +81,8 @@ public class FieldNpc : Actor<Npc> {
 
     public override Stats Stats { get; }
     public int TargetId = 0;
+    public string AiName { get => AiMetadata?.Name ?? ""; }
```

Comment:
I'm not sure if I like this. I get why, as it's only a getter, but the Name for it isn't too much of a hassle to get, as it's right underneath `AiMetadata`

I'd suggest just removing this variable entirely and rely on getting it from AiMetadata.Name whenever you need.